mantidproject/mantid
# Disable ConvFit plot guess checkbox if no resolution file loaded

**Description of work.**
This PR disables the plot guess checkbox within the ConvFit tab if no resolution file has been loaded. This prevents a hard crash.



**To test:**
- Data to test the GUI is in #26631.
- Open the IDA GUI
- Go to the ConvFit tab
- Load Data
- Plot guess should be disabled
- Load resolution file
- Plot guess should be enabled


Fixes #28990  




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
